build-arm.sh

```
@@ -1,42 +1,86 @@
+if [ -f "$HOME/.cargo/env" ]; then
+    . "$HOME/.cargo/env"
 else
-    cargo build $TARGET
+    log_warn "$HOME/.cargo/env not found, continuing..."
 fi
 
-# Copy the built binary to /media/host
-TARGET_DIR="/opt/geckodriver/target"
+TARGET=""
+if [ -n "$HOST_TRIPLE" ]; then
+    TARGET="--target $HOST_TRIPLE"
```

Shellcheck (0.10.0)

[warning] TARGET appears unused. Verify use (or export if used externally).

(SC2034)